src/index.js

```diff
@@ -0,0 +1,233 @@
+const cityNameTextBox = document.getElementById('cityNameInput');
+const tempValue = document.getElementById("tempValue");
+const landscape = document.getElementById("landscape");
+const landscapeImg = document.getElementById("landscapeImg");
+
+// selects
+const skySelect = document.getElementById("skySelect");
+
+// buttons
+const resetCityName = document.getElementById("cityNameReset");
+const convertTempBtn = document.getElementById("convert-temp-btn");
+const increaseTempControl = document.getElementById("increaseTempControl");
+const decreaseTempControl = document.getElementById("decreaseTempControl");
+const currentTempButton = document.getElementById("currentTempButton");
+const citySearchButton = document.getElementById("citySearchButton");
+
+// default values
+const defaultCityNameInput = "Seattle";
+const defaultTempValue = 299.817; // 80 F
+const skies = {
+    chooseAForecast: "&nbsp;",
+    sunny: "☁️☁️☁️☀️☁️☁️☁️",
+    cloudy: "☁️☁️☁️☁️️☁️️☁️",
+    rainy: "🌧🌈🌦🌧🌧💧⛈",
+    snowy: "🌨❄️🌨🌨❄️❄️🌨",
+};
+
+// Keeping one place where we change the temp and unit
+const currentTemperature = {
+    unit: "F",
+    value: defaultTempValue,
+    image: "default.png"
+}
+// Api uses Kelvin convert to F
+const kelvinToF = (k) => {
+    return Math.floor(1.8*(k-273) + 32)
+}
+// Api uses Kelvin convert to C
+const kelvinToC = (k) => {
+    return Math.floor(k - 273.15)
+}
+// Get current temp in the selected unit
+const getTemperatureInCurrentUnit = () => {
+    if(currentTemperature.unit === "F"){
+        return kelvinToF(currentTemperature.value)
+    }
+    return kelvinToC(currentTemperature.value)
+}
+// Get a formatted temp
+const displayTemperature = () => {
+    return `${getTemperatureInCurrentUnit()}°${currentTemperature.unit}`
+}
+
+// Syncs City name between the input box and the title
+const syncCityName =  () => {
+    if(cityNameTextBox.value !== "") {
+        cityNameHeader.textContent = cityNameTextBox.value
+    } else {
+        cityNameHeader.textContent = defaultCityNameInput;
+    }
+    cityNameTextBox.addEventListener("input", (event) => {
+        cityNameHeader.textContent = event.target.value;
+    });
+}
+
+// updates temperature text color and weather garden landscape
+const updateWeatherDisplay = () => {
+    let temp = kelvinToF(currentTemperature.value);
+    if (temp >= 80) {
+        tempValue.style.color = "red";
+        landscape.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+
+    } else if (temp >= 70) {
+        tempValue.style.color = "orange";
+        landscape.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+
+    } else if (temp >= 60) {
+        tempValue.style.color = "yellow";
+        landscape.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+
+    } else if (temp >= 50) {
+        tempValue.style.color = "green";
+        landscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    } else {
+        tempValue.style.color = "teal";
+        landscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    }
```

Comment:
Like you use a data structure for `skies`, you could also use a data structure here to store this info. A list could work here, then you can iterate over the list and check `temp` against the first element of the list and get the color and landscape values. 
```js
temperatureColorsAndLandscapes = [
    [80, "red", "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"],
    [70, "orange", "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"],
    [60, "yellow", "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"],
    [50, "green", "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"],
    [40, "teal", "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"],
]
```